packages/l10n/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "chalk": "2.4.2",
-    "cldr": "5.0.1",
+    "cldr": "5.2.0",
```

Comment:
Can you run the script in the `l10n` package to generate the intl data? Just to make sure that we have up-to-date data.

PS: in #511 I manually used the latest data already, so this version should not change anything in theory.

Reply:
Noted